samples/nrf9160/modem_shell/src/link/link_shell_pdn.c

```diff
@@ -169,13 +177,78 @@ const char *link_pdn_lib_family_to_string(enum pdn_fam pdn_family,
 					 out_fam_str);
 }
 
+static struct link_shell_pdn_info *link_shell_pdn_info_list_add(int pdn_id, uint8_t cid)
+{
+	struct link_shell_pdn_info *new_pdn_info = NULL;
+	struct link_shell_pdn_info *iterator = NULL;
+
+	/* Check if already in list, if there; then return an existing one: */
+	SYS_DLIST_FOR_EACH_CONTAINER(&pdn_info_list, iterator, dnode) {
+		if (iterator->cid == cid) {
+			return iterator;
+		}
+	}
+
+	/* Not already at list, let's add it: */
```

Comment:
Not sure we need colon in the end of each comment. I wouldn't use anything there.

Reply:
Removed those